packages/dds/tree/src/test/feature-libraries/sequence-field/sequenceChangeRebaser.test.ts

```diff
@@ -710,7 +772,7 @@ export function testStateBasedRebaserAxioms() {
 				generateChildStates,
 				fieldRebaser,
 				{
-					groupSubSuites: true,
+					groupSubSuites: false,
```

Comment:
Don't forget to change that back to `true`
Same for the "Tombstone Method" one below